editor/scene/3d/node_3d_editor_plugin.cpp

```diff
@@ -4494,18 +4496,18 @@ void Node3DEditorViewport::assign_pending_data_pointers(Node3D *p_preview_node,
 	accept = p_accept;
 }
 
-void _insert_rid_recursive(Node *node, HashSet<RID> &rids) {
-	CollisionObject3D *co = Object::cast_to<CollisionObject3D>(node);
+void _insert_collision_object_rid_recursive(Node *p_node, HashSet<RID> &p_col_obj_rids) {
+	CollisionObject3D *col_obj = Object::cast_to<CollisionObject3D>(p_node);
```

Comment:
Several renames for clarity, and to match Godot's `p_` naming convention. `node` -> `p_node`, `rids` -> `p_col_obj_rids`, `co` -> `col_obj`, and below `rids` -> `col_obj_rids_to_exclude`.